tox.ini

```diff
@@ -0,0 +1,64 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+[tox]
+skipsdist = true
+
+envlist =
+        {py27}-flake8
+        {py27}-django
+
+
+
+
+[testenv]
+deps =
+        -r{toxinidir}/src/requirements/testing.txt
+        pytest-cov
+        pytest-django
+        pytest-pythonpath
+        pytest-xdist
+
+commands =
+        py.test {posargs}
+
+
+
+
+[testenv:py27-flake8]
+deps =
+        flake8>=2.4,<2.6
+        flake8-import-order<0.8
+
+commands =
+        flake8 src {posargs}
+
+
+
+
+[flake8]
+
+exclude = migrations
+
+application-import-names = luke
+
+
+
+
+[pytest]
+django_find_project = false
+
+python_paths = src/
+
+testpaths = src/
+
+DJANGO_SETTINGS_MODULE = luke.settings.testing
+
+addopts =
+        --doctest-modules
+
+norecursedirs =
+        .*
+        src/requirements
+        src/luke/settings
```

Comment:
Que te parece si automatizamos la parte, de tener que sustituir el nombre "luke", por el nombre del proyecto, algo asi como:

Django development settings for ${PROJECT_NAME} project.

Obviamente lo tendrías que pasar por el mismo lugar donde se le inyectan las variables a los settings y ponerlo en templates, en vez de ponerlo directamente en el proyecto.